features/outlines.yml

```diff
@@ -0,0 +1,22 @@
+name: Outlines
+description: The `outline` CSS property sets the color, style, and width of a line around an element, outside of the border. The `outline-offset` CSS property sets the distance from the element's edge to the outline.
+spec: https://drafts.csswg.org/css-ui-4/#outline
+status:
+  # The `outline` shorthand was introduced at a later date.
+  compute_from: css.properties.outline-style
```

Comment:
I think it's a bit weird for us to name `outline` as the entry point to this feature, yet say that it's supported before the thing we name in the description exists. I think we have three options here (in order of my uninformed feeling about it):

1. `compute_from` on the constituent properties (`outline-color`, `-offset`, etc.) and change the description to match.
2. Break out `outline` into its own feature.
3. Drop the `compute_from` and say that outlines are a recent feature.